Doesn't build on mac

```
$ make
CC main.c
main.c:44:9: warning: missing field 'func' initializer [-Wmissing-field-initializers]
        { NULL }
               ^
main.c:50:8: warning: implicitly declaring library function 'isspace' with type 'int (int)' [-Wimplicit-function-declaration]
        while(isspace(*p))
              ^
main.c:50:8: note: include the header <ctype.h> or explicitly provide a declaration for 'isspace'
main.c:223:8: warning: implicit declaration of function 'whitespace' is invalid in C99 [-Wimplicit-function-declaration]
        while(whitespace(line[i]))
              ^
main.c:277:10: warning: missing field 'has_arg' initializer [-Wmissing-field-initializers]
                { NULL }
                       ^
4 warnings generated.
LD out/xildebug
Undefined symbols for architecture x86_64:
  "_whitespace", referenced from:
      _exec_cmd in main.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [out/xildebug] Error 1
```

😿

--- a/main.c
+++ b/main.c
@@ -1,9 +1,11 @@
+#include <ctype.h>
 #include <getopt.h>
 #include <libusb.h>
 #include <stdio.h>
 #include <readline/history.h>
 #include <readline/readline.h>
 #include <stdbool.h>
+#include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 
@@ -41,7 +43,7 @@ command_t commands[] =
 	{ "quit",			cmd_quit,			"Quit program" },
 	{ "exit",			cmd_quit,			NULL },
 
-	{ NULL }
+	{ NULL, 			NULL,				NULL }
 };
 
 char *strtrim(char *str)
@@ -220,7 +222,7 @@ int exec_cmd(char *line)
 		return -1;
 	}
 
-	while(whitespace(line[i]))
+	while(isspace(line[i]))
 		++i;
 
 	word = line + i;
@@ -274,7 +276,7 @@ int main(int argc, char *argv[])
 	const struct option long_opts[] =
 	{
 		{ "help", no_argument, NULL, 'h' },
-		{ NULL }
+		{ NULL, 0, NULL, 0 },
 	};
 
 	opterr = false;

